sdk/storage/Azure.Storage.Blobs/src/PageBlobClient.cs

```diff
@@ -814,140 +814,128 @@ protected sealed override BlobBaseClient WithSnapshotCore(string snapshot)
                 {
                     content = content.WithNoDispose().WithProgress(progressHandler);
                     var range = new HttpRange(offset, content.Length);
-                    var uploadAttempt = 0;
-                    return await ReliableOperation.DoSyncOrAsync(
-                        async,
-                        reset: () => content.Seek(0, SeekOrigin.Begin),
-                        predicate: e => true,
-                        maximumRetries: Constants.MaxReliabilityRetries,
```

Comment:
Is the `MaxReliabilityRetries` still used anywhere?

Reply:
Yeah it is used in our tests, but probably shouldn't be.